model/src/main/kotlin/spdx/SpdxExpression.kt

```diff
@@ -84,13 +98,18 @@ data class SpdxCompoundExpression(
 data class SpdxLicenseExceptionExpression(
         val id: String
 ) : SpdxExpression() {
+    override fun spdxLicenses(): EnumSet<SpdxLicense> = enumSetOf()
+
     override fun toString() = id
 }
 
 data class SpdxLicenseIdExpression(
         val id: String,
         val anyLaterVersion: Boolean = false
 ) : SpdxExpression() {
+    override fun spdxLicenses(): EnumSet<SpdxLicense> =
```

Comment:
You can drop the return type here in any case.

Reply:
Done.